org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/commands/SurroundWithCommand.java

```diff
@@ -138,9 +141,6 @@ protected Object executeCommand(DOMDocument document, ExecuteCommandParams param
 					startText.append(tag);
 				}
 				startText.append(">");
-				if (emptySelection && snippetsSupported) {

```

Comment:
Do you think this feature isn't helpful? If so, why?

Reply:
Ok I support again this pattern